setup.py

```diff
@@ -87,7 +87,7 @@
         'chardet >=5.1.0, < 6.0',
         'configparser >=5.3.0, < 6.0',
         'humanize >=4.6.0, < 5.0',
-        'openfisca-core >=41.1.0, < 42.0',
+        'openfisca-core @ https://github.com/openfisca/openfisca-core/archive/fix-mypy-checks-periods.zip',
```

Comment:
Warning : Packages installed from PyPI cannot depend on packages which are not also hosted on PyPI.

So this will block PyPi publication of openfisca-survey-manager